_posts/2020-04-01-deep_learning_33_million_with_few_lines_yaml.md

```diff
@@ -80,7 +80,7 @@ The data contains 33,319,019 rows with 15 columns.
 ### **Step 1: Configure YAML - Features**
 
 Now we can build our YAML config. Usually I would use an easy-to-follow
-[configure command](https://docs.zenml.io/docs/developer_guide/pipelines_configure) to create this, but for this post its easier to go section by section and build it manually. So open up a text editor
+[configure command](https://docs.zenml.io/) to create this, but for this post it's easier to go section by section and build it manually. So open up a text editor
```

Comment:
just linking to docs is not really helpful here, should we just remove the link altogether? Or maybe even rewrite this sentence to not mention this command?

Reply:
Removing the link altogether.